setup.py

```diff
@@ -137,8 +137,9 @@ def run_tests(self):
         'show': 'matplotlib',
         'fftw': 'pyfftw',
         'pywavelets': 'Pywavelets',
-        'scikit' : 'scikit-image',
-        'all': test_requires + ['matplotlib', 'pyfftw', 'Pywavelets', 'scikit-image']
+        'scikit': 'scikit-image',
+        'dicom': 'dicom',
```

Comment:
One installs `pydicom` but imports it as [`import dicom`](http://pydicom.readthedocs.io/en/stable/getting_started.html).

So maybe this should be 
`'pydicom': 'dicom'`
or?